File: programmers/lv0/코딩기초트레이닝/조건문/등차수열의특정한항만더하기/Solution.py
def solution(a, d, included):
    x = (a + i * d for i, include in enumerate(included) if include)
    print(*x)
    return sum(a + i * d for i, include in enumerate(included) if include)
    # return answer

x = solution(3, 4, [True, False, False, True, True])
print(x)

File: programmers/lv0/코딩기초트레이닝/조건문/등차수열의특정한항만더하기/Solution.cs
using System;

public class Solution {
    public int solution(int a, int d, bool[] included) {
        int answer = 0;
        for (int i = 0; i < included.Length; i++)
        {
            answer += included[i] ? i * d + a : 0;
        }
        return answer;
    }
}